spec/requests/invoice_items/request_invoice_items_spec.rb

```diff
@@ -35,10 +35,11 @@
 
     get '/api/v1/invoice_items/random'
 
-    invoice_items = InvoiceItem.all.map(&:to_json)
+    invoice_items = InvoiceItem.pluck(:id)
+    body = JSON.parse(response.body)
 
     expect(response).to have_http_status(200)
     expect(response.content_type).to eq('application/json')
-    expect(response.body).to be_in(invoice_items)
+    expect(body['id']).to be_in(invoice_items)
```

Comment:
nice use of this rspec method, I'll have to use that one more often. Also, great job of switching over to an AR method.